### PR TITLE
Add ayatana-indicators.target and systemd dependency

### PR DESCRIPTION
**Pull request by tari01**

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -15,5 +15,8 @@ override_dh_auto_configure:
 		-DCMAKE_INSTALL_DIR=/usr \
 		$(NULL)
 
+override_dh_install:
+	rm -f debian/tmp/usr/lib/systemd/user/ayatana-indicators-pre.target
```
Comment (sunweaver) [inline debian/rules:19]: Why do we need this one?

**Review by sunweaver — CHANGES_REQUESTED** [review]
There is one open question on this.

Furthermore, your commits are two atomic these days for my personal taste. Please reduce this PR to three commits: upstream changes, CI changes, debian/* changes (in that order).

Thanks btw. for adapting debian/*. Much appreciated.

**Comment by sunweaver** [comment]
@tari01: Before I reject your second iteration here, as well. Please incorporate the information I provided via #25 and libayatana-indicator#51 in this PR reworking. Thanks.

**Comment by sunweaver** [comment]
@tari01: Please rebase and amend this PR, so we can soon provide ayatana-indicator-common in libayatana-common. Thanks.

**Comment by tari01** [comment]
> @tari01: Please rebase and amend this PR, so we can soon provide ayatana-indicator-common in libayatana-common. Thanks.

@sunweaver Review time.

**Review by sunweaver — APPROVED** [review]
LGTM now! Thanks!

**Pull request merged**